scripts/mumble-build-number.py

```diff
@@ -9,16 +9,22 @@
 
 import argparse
 import urllib.request
+import urllib.parse
 import sys
 
 def fetch_build_number(commit = None, version = None, password = None):
     if commit is None or version is None:
         return None
 
-    query = "https://mumble.info/get-build-number?commit=" + commit + "&version=" + version
+    parameter = {
+        "commit": commit,
+        "version": version
+    }
 
     if not password is None:
-        query += "&token=" + password
+        parameter["token"] = password
+
+    query = "https://mumble.info/get-build-number?" + urllib.parse.urlencode(parameter)
```

Comment:
Shouldn't the `?` be removed?

Reply:
It's needed to start the parameter list, isn't it? And in the example I found at https://www.programcreek.com/python/example/71124/urllib.parse.urlencode the `?` was also retained :shrug: 
And it definitely works in this form - I tested it